keda/Chart.yaml

```diff
@@ -8,7 +8,8 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.12.0
+
+version: 2.12.1
```

Comment:
Please revert this change, we bump the version during releases

Reply:
reverted